playwright.config.ts

```diff
@@ -19,20 +20,22 @@ const config = {
       name: 'chromium-wide',
       use: {
         ...devices['Desktop Chrome'],
-        viewportSize: { width: 1440, height: 900 },
+        headless: true,
```

Comment:
context? does this break debugging?

Reply:
I had headless set to false while debugging. Default is true, so this isn't a change that will affect anything